### PR TITLE
File scan on sdk v2

### PR DESCRIPTION
**Pull request by pangea-andrest**
Test will fail until FileScan is on prod. Do not merge until then.

### CODE REVIEW

File: examples/file_scan/file_scan_async/src/main/java/cloud/pangeacyber/examples/App.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        try{
+            // Poll result, this could raise another AcceptedRequestException if result is not ready
+            response = client.pollResult(exception.getRequestId(), FileScanResponse.class);
+
+            System.out.println("File Scan poll result success.");
+            FileScanData data = response.getResult().getData();
+            System.out.println("Reputation success");
+            System.out.println("Reputation verdict: " + data.getVerdict());
+            System.out.println("Reputation raw data: " + response.getResult().getRawData());
+		} catch (AcceptedRequestException e) {
```
Comment (govind-salinas-pangea) [inline examples/file_scan/file_scan_async/src/main/java/cloud/pangeacyber/examples/App.java:67]: this formatting seems off

File: examples/file_scan/file_scan_async/src/test/java/cloud/pangeacyber/examples/AppTest.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ */
+public class AppTest
+{
+    /**
+     * Rigorous Test :-)
+     */
+    @Test
+    public void shouldAnswerWithTrue()
+    {
+        assertTrue( true );
```
Comment (govind-salinas-pangea) [inline examples/file_scan/file_scan_async/src/test/java/cloud/pangeacyber/examples/AppTest.java:18]: is this file necessary?

File: examples/file_scan/file_scan_async/src/main/java/cloud/pangeacyber/examples/App.java (hunk 1 above)
Comment (pangea-andrest) [reply examples/file_scan/file_scan_async/src/main/java/cloud/pangeacyber/examples/App.java:67]: Why? How would you do it? It's to show user how to use result and output

File: examples/file_scan/file_scan_async/src/test/java/cloud/pangeacyber/examples/AppTest.java (hunk 2 above)
Comment (pangea-andrest) [reply examples/file_scan/file_scan_async/src/test/java/cloud/pangeacyber/examples/AppTest.java]: Fixed

File: packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/file_scan/ITFileScanTest.java [hunk 3]
```diff
@@ -24,7 +24,7 @@ public class ITFileScanTest {
 	final String TESTFILE_PATH = "./src/test/java/cloud/pangeacyber/pangea/testdata/testfile.pdf";
 
 	FileScanClient client;
-	TestEnvironment environment = TestEnvironment.LIVE;
+	TestEnvironment environment = TestEnvironment.STAGING;
```
Comment (govind-salinas-pangea) [inline packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/file_scan/ITFileScanTest.java:27]: we should not merge SDKs that point to staging i think.

File: examples/audit/log/src/test/java/cloud/pangeacyber/examples/AppTest.java
Comment (govind-salinas-pangea) [inline examples/audit/log/src/test/java/cloud/pangeacyber/examples/AppTest.java:1]: are all these deleted files correct?

**Review by govind-salinas-pangea — COMMENTED** [review]
still some outstanding issues

File: packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/file_scan/ITFileScanTest.java (hunk 3 above)
Comment (pangea-andrest) [reply packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/file_scan/ITFileScanTest.java:27]: No, it was just for testing now, I'll update it once FileScan is on prod

File: examples/audit/log/src/test/java/cloud/pangeacyber/examples/AppTest.java
Comment (pangea-andrest) [reply examples/audit/log/src/test/java/cloud/pangeacyber/examples/AppTest.java:1]: yes

**Comment by pangea-andrest** [comment]
@govind-salinas-pangea all solved now I guess.
